### DISCUSSION TITLE
Is there a way to use x-bind with svg's viewBox without changing the casing?

### DISCUSSION THREAD

**Discussion by fnick851** [alpinejs/alpine#3776] [1. Help]
As seen in this issue: https://github.com/alpinejs/alpine/issues/594
But it seems like this feature is gone. I cannot find it in the documentation.

Basically I need to x-bind:viewBox and preserve the cases in "viewBox".

**Comment by ekwoka** [comment]
That was in Alpine v2.

It could be worth reimplementing this.

**Reply by ekwoka** [reply]
Actually, @fnick851 This should exist now.

`x-bind:view-box.camel`
